src/api/search/server.js

```diff
@@ -0,0 +1,9 @@
+const { Satellite } = require('@senecacdot/satellite');
+
+const query = require('./src/routes/query');
+
+const service = new Satellite();
```

Comment:
You can also do:

```js
const service = new Satellite({ router: query });

Reply:
I tried this, and it didn't seem to find the `/query` route with the above implementation.

Reply:
With 1.5.1 or 1.5.2?

Reply:
I've shipped 1.6.0, fyi.

Reply:
I had tested with 1.5.2, latest at the time. 
Will update to 1.6.0